frontend/src/features/public-form/components/FormPaymentPage/queries.ts

```diff
@@ -34,3 +39,61 @@ export const useGetPaymentInfo = (
     { suspense: true },
   )
 }
+
+const formatCurrency = new Intl.NumberFormat('en-SG', {
```

Comment:
Hmm am wondering if this block here including the validation should be in utils instead of `queries.ts`? as we usually use our queries files for useQuery right? which mostly implies api service calls

Reply:
Great catch. I think it is wrongly placed. Have separated it to `hooks/` and `utils/`

b16ee46ca